### PR TITLE
add 23.12 drop 23.06

### PR DESCRIPTION
**Pull request by karim20230**
Drop reports from ghaf-23.06

    Add reports from ghaf-23.12

### CODE REVIEW

**Comment by karim20230** [comment]
we need to  merge quickly, to avoid conflicts

**Review by henrirosten — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ The Ghaf [vulnerability reports](./reports/) available on this repository are au
 - [Ghaf 'main' generic-x86_64-release](./reports/main/packages.x86_64-linux.generic-x86_64-release.md)
 - [Ghaf 'main' riscv64-linux.microchip-icicle-kit-release](./reports/main/packages.riscv64-linux.microchip-icicle-kit-release.md)
 - [Ghaf 'ghaf-23.09' generic-x86_64-release](./reports/ghaf-23.09/packages.x86_64-linux.generic-x86_64-release.md)
-- [Ghaf 'ghaf-23.06' generic-x86_64-release](./reports/ghaf-23.06/packages.x86_64-linux.generic-x86_64-release.md)
+- [Ghaf 'ghaf-23.12' generic-x86_64-release](./reports/ghaf-23.12/packages.x86_64-linux.generic-x86_64-release.md)
```
Comment (cazfi) [inline README.md:16]: The commit removes ghaf-23.06 directory, but does not create ghaf-23.12 directory, with this .md file in it.

**Review by cazfi — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (henrirosten) [reply README.md:16]: Directory `ghaf-23.12` *should* be automatically generated by the github action, although, I'm not 100% sure if the `git-auto-commit-action` will will pick-up changes to completely new folders. 

We'll see in ~1.5h when this action is ready: https://github.com/tiiuae/ghafscan/actions/runs/7740903556

Comment (karim20230) [reply README.md:16]: yeah I just run the action manually to verify the reports  23.12

Comment (karim20230) [reply README.md:16]: yes  reports 23.12 automatically created! https://github.com/tiiuae/ghafscan/tree/main/reports/ghaf-23.12

**Pull request merged**